services/src/unit-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceTest.java

```diff
@@ -524,4 +472,20 @@ private void setUpSubmitQueryRequest(final ISubmitQueryRequest submitQueryReques
 
     }
 
+    /**
+     * Method to search for message within the Log list
+     * @param logList
+     * @param message
+     * @return Boolean
+     */
+    private static boolean verifyLog(List<ILoggingEvent> logList, String message) {
+        boolean verifyLog = false;
+        for (ILoggingEvent log : logList) {
```

Comment:
Add an exit condition to this loop so that it exits once the message has been found.

Reply:
done